events/syscall/modify_shell_configuration_file.go

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+    "os"
+    "path/filepath"
+
+    "github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+    ModifyShellConfigurationFile,
+    events.WithDisabled(), // this rule is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func ModifyShellConfigurationFile(h events.Helper) error {
+    // Define the path to the file
+    tmpDir := "/tmp"
+    tmpConfigFile := filepath.Join(tmpDir, ".bashrc")
+
+    // Create the file
+    file, err := os.Create(tmpConfigFile)
+    if err != nil {
+        return err
+    }
+    file.Close()
+
+    // Modify the file
+    content := []byte("# written by event-generator\n")
+    err = os.WriteFile(tmpConfigFile, content, 0644)
```

Comment:
`os.WriteFile` already creates the file for us !
Thus we can skip the file creation step ;)